test/functional/toolchains/cmake/cmakedeps2/test_cmakeconfigdeps_new.py

```diff
@@ -1108,7 +1108,7 @@ def build(self):
         assert "protobuf: Release!" in c.out
 
 
-@pytest.mark.tool("cmake", "3.23")
```

Comment:
Do we require `3.27` for CMakeConfigDeps as the minimum-required-version?

Reply:
It is necessary to proof that it works without the CMAKE_MAP_IMPORTED_CONFIGURATIONS for CMake>=3.27